lib/synapse/config_generator.rb

```diff
@@ -0,0 +1,20 @@
+require 'synapse/log'
+require 'synapse/config_generator/base'
+
+module Synapse
+  class ConfigGenerator
+    # the type which actually dispatches generator creation requests
+    def self.create(type, opts)
+      generator = begin
+        type = type.downcase
+        require "synapse/config_generator/#{type}"
```

Comment:
oh i see. yeah, i guess if you `gem install synapse` then it would work with any other systemwide-installed gems. if you install a synapse bundle and `bundle exec synapse`, you have to put the other gem system-wide or bundler won't find it